Banks/creation-banks/creation.css

```
@@ -0,0 +1,239 @@
+* {
+    font-family: "Roboto";
+}
+
+body {
+    background: #f0f2f5;
+}
+
+.title-container {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    border: 1px solid white;
+    padding: 50px;
+    background: white;
+    width: 100px;
+    height: 540px;
+    border-radius: 5px;
+    margin: -10px auto;
+    margin-left: -7px;
+}
+
+.title {
+    text-align: center;
+    position: absolute;
+    top: 8px;
+    margin-left: -28px;
+    transform: translate(-50%, -50%);
+    font-size: 29px;
+    color: green;
+}
+
+.second-title {
+    position: absolute;
+    top: 8px;
+    margin-left: 18px;
+    transform: translate(-50%, -50%);
+    font-size: 29px;
+    color: blue;
+}
+
+.profile {
+    width: 85px;
+    height: 85px;
+    border-radius: 50%;
+    overflow: hidden;
+    margin-top: -570px;
+    margin-left: 37px;
+}
+
+.profile img {
+    width: 85px;
+    height: 85px;
+    margin-top: 1px;
+    margin-left: 1px;
+    object-fit: cover;
+    object-position: center;
+}
+
+.name {
+    align-items: center;
+    position: absolute;
+    margin-top: 10px;
+    margin-left: 25px;
+    font-size: 22px;
+    color: gray;
+}
+
+.bancos {
+    align-items: center;
+    color: gray;
+    font-size: 23px;
+    margin-top: 70px;
+    margin-left: 50px;
+}
+
+.second-container {
+    align-items: center;
+    margin-top: -48px;
+    margin-left: 18px;
+}
+
+.agência {
```

Tirar acento, nunca usar acento ou Ç, nenhuma linguagem suporta. Prefira usar o inglês, não correrá esse risco
```suggestion
.agencia {
```